src/main/java/com/growup/todoary/service/CategoryService.java

```diff
@@ -0,0 +1,25 @@
+package com.growup.todoary.service;
+
+import com.growup.todoary.repository.CategoryRepository;
+import com.growup.todoary.service.dto.response.CategoryResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@Transactional(readOnly = true)
+@RequiredArgsConstructor
+public class CategoryService {
+
+    private final CategoryRepository categoryRepository;
+
+    public List<CategoryResponse> findAll() {
+        return categoryRepository.findAll()
+                .stream()
+                .map(CategoryResponse::new)
+                .collect(Collectors.toList());
```

Comment:
java 16에서 제공하는 Stream.toList() 메소드가 있다는 사실 알고계셨나요~?

```suggestion
                .toList();
```

반드시 적용할 필요는 없지만 다음과 같은 특징이 있다고하는데 한번 확인해보시길 바래요

<img width="671" alt="image" src="https://github.com/grow-up-study/todoary/assets/49794401/9173caa2-fa11-4b0f-b7dc-bc3ce098b38c">

[참고자료](https://stackoverflow.com/questions/65969919/differences-of-java-16s-stream-tolist-and-stream-collectcollectors-tolist)

Reply:
몰랐습니다..! 좀 더 찾아보겠습니당!!